src/Function.cpp

```diff
@@ -397,6 +444,7 @@ Function::Function(const string &name, const Type *return_type)
 	FuncList.push_back(this);			// Add to global list of functions.
 	func_attr_inline = false;
 	func_attr_aligned = false;
+	common_func_attributes = {"artificial", "flatten", "no_reorder", "hot", "cold", "no_ipa", "no_throw", "deprecated", "no_icf", "no_profile_instrument_function", "no_instrument_function", "no_sanitize_address", "no_sanitize_thread", "no_sanitize_undefined", "no_split_stack", "noinline", "noplt", "stack_protect"};
```

Comment:
Do not create this variable, make it a local variable in `Function::GenerateAttributes`.

Reply:
Sure.